src/test/java/test/org/jenkinsci/plugins/workinghours/WorkingHoursQueueTaskDispatcherTest.java

```diff
@@ -158,6 +158,9 @@ public void testCanRunBlocked() {
         assertNotNull(instance.canRun(item));
     }
 
+
```

Comment:
Can you delete the extra blank lines so the spacing's consistent with the reset of the file?